<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-O6A5C2O" failures="3" tests="9" name="IntroMessage Announcement Update" time="390.701" errors="0" timestamp="2019-11-15T16:08:03 IST">
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementUpdateTest" name="EditCancelIntroMessageAnnouncementRecord" time="8.246">
    <failure type="java.lang.AssertionError" message="Edit Cancel assertion Failed did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Edit Cancel assertion Failed did not expect to find [true] but found [false]
at com.tetherfi.test.ivr.IntroMessageAnnouncementUpdateTest.EditCancelIntroMessageAnnouncementRecord(IntroMessageAnnouncementUpdateTest.java:75)
at java.base/java.util.ArrayList.forEach(Unknown Source)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- EditCancelIntroMessageAnnouncementRecord -->
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementUpdateTest" name="EditRecordWithoutModifyReason" time="9.977"/>
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementUpdateTest" name="EditRevertIntroMessageAnnouncementRecord" time="9.78"/>
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementUpdateTest" name="VerifyRevertForEditRecord" time="10.081"/>
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementUpdateTest" name="VerifyAuditTrialReportForRevertUpdate" time="21.1">
    <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at com.tetherfi.test.ivr.IntroMessageAnnouncementUpdateTest.VerifyAuditTrialReportForRevertUpdate(IntroMessageAnnouncementUpdateTest.java:120)
at java.base/java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- VerifyAuditTrialReportForRevertUpdate -->
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementUpdateTest" name="EditRejectRecord" time="9.569"/>
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementUpdateTest" name="VerifySendForApprovalForEditRejectRecord" time="9.933"/>
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementUpdateTest" name="RejectforEditIntroMessageAnnouncementRecord" time="8.487"/>
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementUpdateTest" name="VerifyAuditTrailReportForReject" time="22.933">
    <failure type="java.lang.AssertionError" message="Audit Trail report assertion failed did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Audit Trail report assertion failed did not expect to find [true] but found [false]
at com.tetherfi.test.ivr.IntroMessageAnnouncementUpdateTest.VerifyAuditTrailReportForReject(IntroMessageAnnouncementUpdateTest.java:162)
at java.base/java.util.ArrayList.forEach(Unknown Source)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- VerifyAuditTrailReportForReject -->
</testsuite> <!-- IntroMessage Announcement Update -->
